test/test_optim.py

```diff
@@ -597,6 +597,14 @@ def _compare_between(self, inputs, models, optimizers, assert_eq_kwargs=None, as
             for input, model, optimizer in zip(inputs, models, optimizers):
                 optimizer.zero_grad()
 
+                if i == 4:
```

Comment:
Hmm this should work, but maybe try freezing earlier on + bringing it back after 2 iters instead of 1. Maybe the discrepancy is too small?

Reply:
sure, i also move the np-op test to 2nd iter, so the last (6th) iter can run the whole model.